### PR TITLE
The option to overwrite the table names now made effect

### PR DESCRIPTION
**Pull request by ricmalta**
I was not able to use custom table names to co-exist with my current DB naming style, with this small fix it does.

### CODE REVIEW

**Comment by yonjah** [comment]
What is your use case ?
I don't think allowing totally override of the table names is a good idea since it can be very confusing.
The prefix there to prevent collisions in case you already have a table with the same name in your database.

**Comment by ricmalta** [comment]
My use case is exactly to add the remaining tables to my existent DB.
I use camel case in table naming and without this fix it's not possible to have it.
In your unit tests you have a test to overwrite the table naming by passing an object, which simply won't work with the array iteration in the implementation. What I did was simply to fully pass the test by using the object value instead of the key.

**Comment by yonjah** [comment]
Ok I can see your point but there are a few things you need to consider.

The test suite was a copy past from `node_acl` as that's what they recommended for testing new backends.
If you actually run the test suite on the original code all tests pass.
I can agree that's a useless test but it wont fail. 

So if you want to really address the issue you have to rewrite the test and add some more tests (just checking the options is useless you need to check data actually goes into and fetched from the correct bucket using those settings).

That goes without saying that you need to actually run the full test suite and see how the code behaves.

Once this is done you should also update the documentation to mention how to use this option 

It would probably make sense to remove the prefix options since I don't think there is reason for having both.

Just keep in mind that I don't really have intention of maintaining this module as I'm not currently using `node_acl`. I will accept proper pull requests but I think this module has more serious issues and I wouldn't use it in production as is

File: lib/dbTasks.js [hunk 1]
```diff
@@ -25,11 +25,12 @@ module.exports = {
 			};
 
 		buckets.forEach(function (table) {
-			var name = prefix + table;
+			var name = prefix + options.buckets ? options.buckets[table] : table;
```
Comment (yonjah) [inline lib/dbTasks.js]: This is equal  to 
`var name = (prefix + options.buckets) ? options.buckets[table] : table;`
Which is not really what you want here.

Comment (ricmalta) [inline lib/dbTasks.js]: True, it's a bad implementation!

**Comment by ricmalta** [comment]
I'll have your requirements in consideration. Makes sense to do those so.

I thought about starting a new node acl driver, but I think it's better to support an existing one!

**Comment by yonjah** [comment]
I don't know. it was a few hours writing this module, just to see if `node_acl` sould fit my use case and it turned out it didn't. I'm sure that some of the code here can be used as baseline for a `Sequelize` module but some TLC is needed.

Also I'm not sure if `node_acl` is really built to play nicely with relational databases it seems like the original design is highly focused around key-value storage. I wouldn't use mongodb but if you keen on using node_acl maybe having some complementary key-value storage complementary to your main db would be beneficial

**Pull request closed without merging**